app/helpers/bookmarks_helper.rb

```diff
@@ -71,17 +71,13 @@ def get_symbol_for_bookmark(bookmark)
   end
 
   def bookmark_form_path(bookmark, bookmarkable)
-    if bookmark && bookmark.new_record?
-      return "" unless bookmarkable
-      case bookmarkable.class.to_s
-      when "Work"
-        work_bookmarks_path(bookmarkable)
-      when "ExternalWork"
+    if bookmark.new_record?
+      if bookmarkable.new_record?
         bookmarks_path
```

Comment:
Is this the case where people make a new bookmark on a new external work?

Reply:
Yes, this is for bookmarks of a new external work. Everything else uses a nested resource path instead of passing `bookmarkable_type`/`bookmarkable_id` (which should hopefully help prevent issues like [AO3-6001](https://otwarchive.atlassian.net/browse/AO3-6001)).